db/migrations/20240517140534-avoid-tabs-with-postgres.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const _ = require('lodash');
+
+// Fix remediation plan names that have leading/trailing tabs
+//
+// Something about the combination of Sequelize and postgres makes sql containing \t's behave unpredictably,
+// so we're doing this migration again, the tedious way...
+exports.up = async (q) => {
+    // get list of remediation plans
+    const [plans, metadata] = await q.sequelize.query("SELECT name, tenant_org_id, id FROM remediations;");
```

Comment:
Could this return many records that could lead to a memory limit hit?

Reply:
Yeah, this was sort of "work in progress" and is deficient in several respects - updated code coming shortly!